ddtrace/internal/import_hooks/decorator.py

```diff
@@ -0,0 +1,38 @@
+from .registry import hooks
```

Comment:
Opinion, but I feel like there's a lot of tiny files. I'd be fine with having just `import_hooks.py` and the whole code, that'd be more readable than jumping around 3 files that are 8 or 38 lines long :)

Reply:
micro-modules....